static/src/javascripts/projects/common/modules/analytics/discussion.js

```diff
@@ -106,21 +106,20 @@ define([
     // Convenience functions
     track.areCommentsSeen = function () {
         var timer,
-            scroll = function () {
-                if (!track.seen && !timer && track.areCommentsVisible()) {
+            scroll = function (scrollY) {
+                if (!track.seen && !timer && track.areCommentsVisible(scrollY)) {
                     track.scrolledToComments();
-                    mediator.off('window:scroll', _.debounce(scroll, 200));
+                    mediator.off('window:throttledScroll', scroll);
```

Comment:
this is degrading performances. The original code was throttling scroll events for 200ms, 'window:throttledScroll' is throttled by one single frame, I think you still want to `debounce` this and similar listeners